src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -845,11 +849,11 @@ internal sealed class AssemblyInfo
         internal MetadataReader asmMetadataReader { get; }
         internal MetadataReader pdbMetadataReader { get; set; }
 
-        internal List<Tuple<MetadataReader, MetadataReader>> enCMetadataReader  = new List<Tuple<MetadataReader, MetadataReader>>();
+        internal List<Tuple<MetadataReader, MetadataReader>> enCMetadataReader = new();
```

Comment:
Sorry, one more:

```suggestion
        internal List<(MetadataReader asm, MetadataReader pdb)> enCMetadataReader = new();
```

This will ensure that we don't mistakenly get the wrong member.